modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -13611,7 +13611,7 @@ position.third=Third
 possible-assignees-must-have-permissions-to-be-assigned-to-the-corresponding-step=Possible assignees must have permissions to be assigned to the corresponding step.
 post=Post
 post-answer=Post Answer
-post-api-endpoints-retrieve-type-must-be-x=POST API endpoints retrieve type must be "${0}."
+post-api-endpoints-retrieve-type-must-be-x=POST API endpoints retrieve type must be "{0}."
```

Comment:
I think the dot shouldn't be there. It should be `post-api-endpoints-retrieve-type-must-be-x=POST API endpoints retrieve type must be "${0}".`

Reply:
Thanks @jalber786 I think so too, but Brian told that the dot should be inside and here [is the link](https://writeshop.com/blog/how-to-use-quotation-marks/) he shared before 
![image](https://github.com/liferay-headless/liferay-portal/assets/1308942/9c2bd72b-e311-48d7-a184-3a43e40d8c22)